bin/node/bench/src/core.rs

```diff
@@ -126,18 +127,22 @@ pub fn run_benchmark(benchmark: Box<dyn BenchmarkDescription>, mode: Mode) -> Be
 	let name = benchmark.name().to_owned();
 	let mut benchmark = benchmark.setup();
 
-	let mut durations: Vec<u128> = vec![];
-	for _ in 0..50 {
-		let duration = benchmark.run(mode);
-		durations.push(duration.as_nanos());
-	}
+	let durations: Vec<u128> = (0..50)
+		.into_par_iter()
```

Comment:
These benches run serially since they otherwise influence each other.  

Also this code is pretty dated and should eventually be removed (https://github.com/paritytech/substrate/issues/12218).